typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerRouterDeployer.ts

```diff
@@ -64,30 +66,38 @@ export class LiquidityLayerDeployer extends MiddlewareRouterDeployer<
     super(multiProvider, configMap, liquidityLayerFactories, create2salt);
   }
 
+  routerContractName(): string {
+    return 'liquidityLayerRouter';
```

Comment:
Move to a const? Odd use of class method imho

Reply:
Very odd, agreed